src/oas.md

```diff
@@ -475,7 +478,7 @@ An object representing a Server.
 
 | Field Name | Type | Description |
 | ---- | :----: | ---- |
-| <a name="server-url"></a>url | `string` | **REQUIRED**. A URL to the target host. This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the document containing the Server Object is being served. Variable substitutions will be made when a variable is named in `{`braces`}`. |
+| <a name="server-url"></a>url | `string` | **REQUIRED**. A URL to the target host. This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the document containing the Server Object is being served. Variable substitutions will be made when a variable is named in `{`braces`}`.   Note that the [OpenAPI Object's](#openapi-object) `$self` field is **NOT** used for relative URL reference resolution. |
```

Comment:
But we do use the document's URI itself -- so should we mention that explicitly here?

Reply:
I'm not sure if what you're thinking is the same as what I'm thinking. Can we come up with some examples?

Reply:
@karenetheridge yes, see other comments on how I did write examples, they just got lost on another branch.  I will update as soon as the house-moving tasks let up, which will probably be next week sometime.

Reply:
> This indicates that we use the document's URL, but not its $self-defined URI. I'm uncertain if a.) this is the right approach, or b.) it is explained well.

(b) was indeed the case above. Your examples helped, as per my new comments now.